src/Microsoft.Azure.ServiceBus/Management/QueueDescription.cs

```diff
@@ -0,0 +1,329 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Azure.ServiceBus.Primitives;
+
+namespace Microsoft.Azure.ServiceBus.Management
+{
+    // TODO: Input validation
+    public class QueueDescription : IEquatable<QueueDescription>
```

Comment:
Looks like de-dup related properties are missing

Reply:
Nope. Its present. Ctrl+F "Duplicate"

---
In reply to: [194202383](https://github.com/Azure/azure-service-bus-dotnet/pull/481#discussion_r194202383) [](ancestors = 194202383)